Draft.md

```diff
@@ -174,19 +174,19 @@ Wie bei der Besprechung einer wissenschaftlichen Publikation ist die Zusammenset
+* **Sind die Algorithmen korrekt implementiert worden?** Die Korrektheit der Ergebnisse einer Software ist mitunter schwer zu beurteilen. Insbesondere dann, wenn es keine weiteren Implementierungen der Funktionalität in anderen Softwarepaketen gibt. Werden die in der Dokumentation genannten Algorithmen fehlerfrei in Programmcode ausgedrückt? Ist das wissenschaftliche Ergebnis jenseits aller technischen Unterschiede vergleichbar zu anderen Implementierungen oder Werkzeugen? Sind die verwendeten Algorithmen dokumentiert und hinreichend wissenschaftlich belegt? Auch diese Frage mag im Rahmen einer Rezension nicht abschließend zu beantworten sein.
 
-* **Gibt es Archäologie-relevante Projekte/Anwendungen, in der diese Software bereits angewendet wurden?** Wie plausibel sind die erzielten Resultate? Wie wurden sie von der wissenschaftlichen Community aufgenommen?
+* **Gibt es Archäologie-relevante Projekte/Anwendungen, in der diese Software bereits angewendet wurden?** Software wird oft von und für Forschungsprojekte entwickelt. Zur Beurteilung ihrer Qualität und Relevanz kann es also mitunter hilfreich sein, die Forschungsprojekte selbst genauer anzusehen. Wie plausibel sind die erzielten Resultate, die mit der Software zusammenhängen? Wie wurden sie von der wissenschaftlichen Community aufgenommen?
 
-* **Wie ist die Software publiziert?** Eignet sich die Software für wissenschaftliche Anwendungen, da sie in einer zitierbaren Publikation vorgestellt wurde? Sind einzelne Versionen klar als solche referenzierbar (etwa mittles DOI)? z.B. "Das Softwarepaket wurde in einer kurzen, zweiseitigen Publikation im Journal *Open Source Software* vorgestellt, diese Publikation schließt aber keine umfassenden Tests oder Vergleiche mit Alternativprodukten ein"
+* **Wie ist die Software publiziert?** Idealerweise sollte Software für wissenschaftliche Anwendungen auch wissenschaftlich publiziert sein. Das macht sie einerseits leichter zitierbar und bedeutet gegebenenfalls auch, dass die Software einen Peer Review Prozess durchlaufen hat. Gibt es ein Benchmarking-Paper, dass das Werkzeug explizit vorstellt und mit alternativen Produkten vergleicht? Sind einzelne Versionen der Software klar als solche referenzierbar (etwa mittles DOI)? Hat das Journal in dem veröffentlicht wurde, einen expliziten Software-Peer-Review Mechanismus?
 
```

Comment:
Was ist ein Benchmarking-Paper ? Können "Forschungssoftware" und "Werkzeug" synonym gesetzt werden? Spielen solche Journale, mit Software-Peer-Review eine Rolle in dem Sektor in dem archäologische Forschungssoftware genutzt wird ? Ich will die Punkte nicht in Abrede stellen, aber dazu anregen, sie als eine Spielart in einem sehr breiten Spektrum zu sehen.

Reply:
Zunächst: Diese Fragen in den Stichpunkten können meiner Ansicht nach nicht alles abdecken, sondern nur bestimmte Perspektiven anregen.

Ein Benchmarking Paper legt offen, wie die Software im Vergleich zu anderer Software abschneidet. Oft wird Benchmarking von den Autoren einer neuen Software selbst bereitgestellt, um ihr neues Tool zu feiern. Hier ist zum Beispiel eines, dass wir mal in der Arbeitsgruppe besprochen haben: https://www.biorxiv.org/content/10.1101/2020.02.06.938126v2.full 
Es gibt auch unabhängige Vergleichspaper. Hier zum Beispiel ein richtig cooles, dass neulich wichtig für mich war: https://link.springer.com/article/10.1007/s13253-018-00348-w

Forschungssoftware und Werkzeug sind hier synonym. Aber gelegentlich sollte man ja auch ein anderes Wort einsetzen :smile: 

Ich glaube das CAA Journal macht auch Software-Peer-Review, bin aber nicht sicher.